app/lib/screens/faq/Item.dart

```diff
@@ -15,16 +16,357 @@ class FaqItemScreen extends StatefulWidget {
+        backgroundColor: Colors.black,
+        elevation: 0,
+        actions: <Widget>[
+          IconButton(
+            onPressed: () {},
+            icon: const Icon(Icons.menu),
+          )
+        ],
+      ),
+      backgroundColor: Colors.black,
+      body: Column(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        children: <Widget>[
+          Padding(
+            padding: const EdgeInsets.fromLTRB(8.0, 8.0, 8.0, 8.0),
+            child: Column(
+              children: [
+                Padding(
+                  padding: const EdgeInsets.fromLTRB(12.0, 4.0, 12.0, 4.0),
+                  child: Row(
+                    children: [
+                      Flexible(
+                        child: Text(
+                          widget.faq.title(),
+                          style: GoogleFonts.roboto(
+                            color: Colors.white,
+                            fontSize: 16,
+                            fontWeight: FontWeight.w600,
+                          ),
+                        ),
+                      ),
+                    ],
+                  ),
+                ),
+                Padding(
+                  padding: const EdgeInsets.only(top: 12.0, bottom: 12.0),
+                  child: Row(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    children: [
+                      Container(
+                        height: 40.0,
+                        padding: const EdgeInsets.all(8.0),
+                        margin: const EdgeInsets.only(left: 12.0),
+                        decoration: BoxDecoration(
+                          borderRadius: BorderRadius.circular(10),
+                          color: Colors.grey[800],
+                          border: Border.all(
+                            color: Colors.white,
+                          ),
+                        ),
+                        child: Row(
+                          crossAxisAlignment: CrossAxisAlignment.start,
+                          children: [
+                            Image.asset(
+                              'assets/images/asakerImage.png',
+                            ),
+                            Padding(
+                              padding: const EdgeInsets.only(
+                                left: 8.0,
+                              ),
+                              child: Text(
+                                'Support',
+                                style: GoogleFonts.roboto(
+                                  color: Colors.white,
+                                  fontSize: 16,
+                                ),
+                              ),
+                            )
+                          ],
+                        ),
+                      ),
+                      const Spacer(),
+                      SizedBox(
+                        height: 40,
```

Comment:
I don't think we always want to show the comments... let's hide this area at first and only show upon press of the comments-button.